### DISCUSSION TITLE
You've used 75% of included services for GitHub Actions.

### DISCUSSION THREAD

**Discussion by thomasmerz** [upptime/upptime#261] [Show and tell] [answered]
Details: https://docs.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/managing-your-spending-limit-for-github-actions#about-spending-limits-for-github-actions

> GitHub Actions usage is free for both public repositories and self-hosted runners. For private repositories, each GitHub account receives a certain amount of free minutes and storage, depending on the product used with the account.

Is this per month? Well, this could be ok for some private accounts with no "too frequently" running GH Actions or with no more other repos with GH Actions… 🤔

@AnandChowdhary , is this worth to be documented regarding usage of upptime with "GitHub Free" Plans?

**Comment by AnandChowdhary** [comment]
I don't understand, what would like like me to document? You'll only see the message "You've used 75% of included services" when you're using a private repository since public repositories get unlimited build minutes.

Yes, perhaps it's a good idea to let people know that Upptime would end up using hundreds (if not thousands) of minutes every month, and it's recommended to use a public repository unless you understand that you'll have to bare the expenses of the used build minutes in private repositories.

**Comment by AnandChowdhary** [comment, marked as answer]
I've added a billing note to the "Get started" page: https://github.com/upptime/upptime.js.org/commit/3dab6b235c1265b75db24c8c3f6a03cea86dc735:

> **Billing note:** Upptime uses thousands of build minutes every month (approximately 3,000 minutes in the default setting). If you use a public repository, GitHub offers unlimited free build minutes, but if you use a private repository, you'll have to pay for this time.
